resources/init-default.js

```diff
@@ -159,5 +159,7 @@ module.exports = {
     "email": config['init.author.email'],
     "url": config['init.author.url']
   } : undefined,
-  "license": 'MIT'
+  "license": 'MIT',
+  "repository": "www.github.com/tessel/t2-cli",
```

Comment:
0_o This is a duplicate https://github.com/tessel/t2-cli/blob/jon-fix-init-pkg-json/resources/init-default.js#L134-L154